### PR TITLE
add fused_seqpool_cvm op

### PR DESCRIPTION
**Pull request by danleifeng**
### PR types

Performance optimization
### PR changes

OPs
### Describe

add fused_seqpool_cvm op for performance

### CODE REVIEW

File: python/paddle/fluid/contrib/layers/nn.py [hunk 1]
```diff
@@ -537,6 +537,64 @@ def fused_embedding_seq_pool(input,
...
+    This layer is the fusion of sequence_pool and continuous_value_model.
+
+    **Notes: The Op only receives List of LoDTensor as input, only support SUM pooling now.
+
+    Args:
+        input(Variable|list of Variable): Input is List of LoDTensor.
+        pool_type(str): pooling type, only support SUM pooling now.
+        cvm(Variable): cvm Variable.
+        pad_value(float): padding value of sequence pool.
+        use_cvm(bool): use cvm or not.
```
Comment (TCChenlong) [inline python/paddle/fluid/contrib/layers/nn.py:558]: Please add the information of `cvm_offset ` ，and default value of `pad_value` and `use_cvm ` .

File: python/paddle/fluid/contrib/layers/nn.py [hunk 2]
```diff
@@ -537,6 +537,64 @@ def fused_embedding_seq_pool(input,
...
+    Returns:
```
Comment (TCChenlong) [inline python/paddle/fluid/contrib/layers/nn.py:559]: Please add a  blank line before **`Returns`** .

File: python/paddle/fluid/contrib/layers/nn.py [hunk 3]
```diff
@@ -537,6 +537,64 @@ def fused_embedding_seq_pool(input,
...
+        Variable|list of Variable: The tensor variable storing sequence pool and cvm
+        of input.
```
Comment (TCChenlong) [inline python/paddle/fluid/contrib/layers/nn.py:561]: Please add  code example.

File: python/paddle/fluid/contrib/layers/nn.py (hunk 1 above)
Comment (danleifeng) [reply python/paddle/fluid/contrib/layers/nn.py:558]: will update in the next PR,thanks

File: python/paddle/fluid/contrib/layers/nn.py (hunk 2 above)
Comment (danleifeng) [reply python/paddle/fluid/contrib/layers/nn.py:559]: will update in the next PR,thanks

File: python/paddle/fluid/contrib/layers/nn.py (hunk 3 above)
Comment (danleifeng) [reply python/paddle/fluid/contrib/layers/nn.py:561]: will update in the next PR,thanks

**Review by zhiqiu — APPROVED** [review]
LGTM for static only api, by the way, you can add `:api_attr: Static Graph`

**Comment by danleifeng** [comment]
> :api_attr: Static Graph

thanks, will update in the next pr

**Review by Aurelius84 — APPROVED** [review]
LGTM for data_type registeration

**Review by XiaoguangHu01 — APPROVED** [review]
LG API

**Review by TCChenlong — APPROVED** [review]
LGTM
TODO：Fix API docs

**Pull request merged**